### PR TITLE
Defect-123

### PR DESCRIPTION
**Pull request by AkashGitRepo123**
fixed compilation issue

### CODE REVIEW

**Comment by AkashGitRepo123** [comment]
Hi  @satishsamp,

Could you please review the code?

**Review by satishsamp — COMMENTED** [review]
Please check msg initialization is correct.

File: src/com/methodreference/ConstructorReference.java [hunk 1]
```diff
@@ -3,8 +3,8 @@
 public class ConstructorReference {
 	public static void main(String[] args) {
 
-		/*Messageable hello =*/ Message::getMsg;
```
Comment (satishsamp) [inline src/com/methodreference/ConstructorReference.java:6]: Hi Please check compilation error.

File: src/com/methodreference/ConstructorReference.java [hunk 2]
```diff
@@ -15,7 +15,7 @@ public static void main(String[] args) {
 
 class Message {
 
-	String msg;
+	String msg = "hi";
```
Comment (satishsamp) [inline src/com/methodreference/ConstructorReference.java:18]: This value needs to be changed to Hello.

**Review by satishsamp — CHANGES_REQUESTED** [review]
Please do the changes and let me know.

**Pull request merged**